app/views/modules/_header.html.erb

```diff
@@ -13,26 +13,28 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-  <div class="custom-header">
-    <div class="row">
-      <div class="row-fluid whitespace-half">
-      <header>
-        <div class="col-sm-2 logo">
-        <%= link_to root_path do %>
-          <%= image_tag "Home.png", alt: "Avalon Media System - #{t(:release_label)} #{Avalon::VERSION}" %>
-        <% end %>
-        </div>
-        
-        <!-- <div class="col-md-6">
-          <h1>Avalon Media System</h1>
-          <p class="lead"><%=release_text%></p>
-        </div>    -->
-
-        <%# By default shift log in / log out to the header %>
-        <div class="pull-right log-in-out">
-          <%= render 'modules/user_management' %>
-        </div>
-      </header>
+
+ <div class="custom-header" id="masthead">
+
+  <div class="row">
+    <div class="row-fluid whitespace-half">
+    <header>
+
+      <div class="lib-logo col-sm-5 hidden-xs">
+        <%= ualib_logo_link %>
+        <%= link_to "ERA Home", "https://era.library.ualberta.ca", :class => "era-link" %>
+      </div>
+
+      <div class="mast-branding col-sm-4">
+        <%= era_av_logo_link %>
+      </div>
+
+      <%# By default shift log in / log out to the header %>
+      <div class="col-sm-3">
+        <%= render partial: "modules/user_util_dropdowns"%>
```

Comment:
The partial isn't that big, and this file itself isn't that big. Could just plop the code right here and get rid of the extra partial